composables/transaction/transactionOffer.ts

```diff
@@ -41,7 +40,7 @@ async function execMakingOffer(item: ActionOffer, api, executeTransaction) {
           amount: Number(price) || 0,
           direction: 'Send',
         },
-        duration,
+        300 * 24 * duration, // 12sec /block --> 300blocks/hr
```

Comment:
move alongside the variable / remove comment block 


```suggestion
        BLOCKS_PER_HOUR * 24 * duration,  // 12sec /block --> 300blocks/hr
```

Reply:
updated